### DISCUSSION TITLE
CLI builder to use with Deno

### DISCUSSION THREAD

**Discussion by d9k** [vanilla-extract-css/vanilla-extract#1198] [Ideas]
The are growing number of users choosing [Deno](https://deno.com) frameworks: [ultra](https://ultrajs.dev), [aleph](https://alephjs.org), [fresh](https://fresh.deno.dev).

Please make simple CLI builder to extract css from `*.css.ts` files.

Something like

```
vanilla-extract-cli [--watch] import_scripts_folder explort_css_file_path
```

I guess Deno developers can 
1) Use importMap to map `@vanilla-extract/css` to actual `https://esm.sh/@vanilla-extract/css@1.13.0`.
2) Somehow run parcel on separate node process with [parcel plugin](https://vanilla-extract.style/documentation/integrations/parcel) only: 

```
  "transformers": {
    "*.css.ts": ["@vanilla-extract/parcel-transformer"]
  }
```

But minimum parcel-on-node-js dependencies

```
  "dependencies": {
    "@vanilla-extract/css": "^1.13.0",
    "@vanilla-extract/parcel-transformer": "^1.0.0",
    "parcel": "^2.10.0"
  }
```

installed with `yarn` are whopping 380 mb!

I don't think Deno users would bother installing such bloated enviromnent alongside with the project :pensive:

**Comment by askoufis** [comment]
This isn't really the fault of Vanilla Extract, but rather `parcel`. [Its install size is 354mb](https://pkg-size.dev/parcel).

If you want a lighter-weight way to extract CSS from your `css.ts` files, you could try using a smaller bundler like `rollup` instead to create a library containing your CSS files.
